### PR TITLE
Allow people to share links to the playground

### PR DESCRIPTION
**Pull request by ChayimFriedman2**
We do that by encoding the code (compressed) in the URL.

The compression is the same used in the TypeScript playground:
https://github.com/microsoft/TypeScript-Website/blob/e9d8f66f6b8be2dda06737d3686dcb795749dff2/packages/sandbox/src/compilerOptions.ts#L119-L120

### CODE REVIEW

**Comment by ruby0x1** [comment]
Feel free to experiment with that. The code for the [try page is here ](https://github.com/wren-lang/wren/blob/main/doc/site/static/wren.js)

**Comment by joshgoebel** [comment]
Yes, this would be pretty awesome and I think anyone could probably contribute it... just a "share" button that generates a URL you can copy - the entire source being encoded into the URL... same way it seems TS does it.

**Comment by ruby0x1** [comment]
Oh, I misread this as an issue but I see it's a PR. I'll look into it soon, thanks!

File: doc/site/static/wren.js [hunk 1]
```diff
@@ -34,6 +36,21 @@ window.onload = function() {
       jar.updateCode(content);
     }
 
+    share.onclick = (e) => {
+      var code = jar.toString()
+      var compressed = LZString.compressToEncodedURIComponent(code)
+      var url = location.protocol + "//" + location.host + location.pathname + "?code=" + compressed
```
Comment (joshgoebel) [inline doc/site/static/wren.js:42]: Perhaps use a template literal here?

**Review by joshgoebel — COMMENTED** [review]
I dunno about the UI but I just skimmed the code, looks like about what I'd expect.

**Comment by ruby0x1** [comment]
Thanks @ChayimFriedman2 that's a great starting point and a very helpful addition (especially without server side). It should be live soon

File: doc/site/static/wren.js (hunk 1 above)
Comment (ChayimFriedman2) [reply doc/site/static/wren.js:42]: The code pretty much uses ES5 so I keeped it so.

**Pull request merged**